packages/react-storage/src/components/StorageBrowser/Views/Controls/BreadcrumbsControl.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { withBaseElementProps } from '@aws-amplify/ui-react-core/elements';
+
+import { StorageBrowserElements } from '../../context/elements';
+
+const { Button, ListItem, Nav, OrderedList, Span } = StorageBrowserElements;
+
+const BLOCK_NAME = `breadcrumbs`;
+
+export interface BreadcrumbItem {
+  label?: string;
+}
+
+export interface BreadcrumbsControlProps {
+  items?: BreadcrumbItem[];
+}
+
+export interface BreadcrumbsControl<
+  T extends Partial<StorageBrowserElements> = StorageBrowserElements,
+> {
+  /* ???? Where/how should we define these top level props */
+  ({ items }: BreadcrumbsControlProps): React.JSX.Element;
```

Comment:
We shouldn't be reading these values as props since they are tracked in the `HistoryState` but this is fine until we integrate